test/rulePriorities.js

```diff
@@ -71,53 +71,51 @@ describe('Rule Priorities', () => {
         assert.ok(TRACKER_BLOCKING_BASELINE_PRIORITY < TRACKER_BLOCKING_CEILING_PRIORITY)
         assert.ok(TRACKER_ALLOWLIST_BASELINE_PRIORITY < TRACKER_ALLOWLIST_CEILING_PRIORITY)
 
-        // Tracker allowlist should take priority over tracker blocking, but not
+        // Tracker allowlist should take priority over tracker blocking and smarter encryption, but not
         // other features/allowlists.
+        assert.ok(TRACKER_ALLOWLIST_CEILING_PRIORITY > SMARTER_ENCRYPTION_PRIORITY)
```

Comment:
Should this be testing the tracker baseline priority?

Reply:
Yes, good catch.